css/src/components/persona.scss

```diff
@@ -0,0 +1,82 @@
+$persona-avatar-dimensions: 16 24 32 36 48 !default;
+$persona-avatar-background-color: $alternate-background-medium !default;
+$persona-avatar-font-color: $text-invert !default;
+$persona-default-size: 32px !default;
+$persona-font-size: $font-size-7 !default;
+$persona-gap-size: $layout-1 !default;
+$persona-metadata-font-size: $font-size-9 !default;
+$persona-metadata-font-color: $text-subtle !default;
+$persona-dimension-font-size-ratio: 0.45;
+
+.persona {
+	display: flex;
+	position: relative;
+	gap: $persona-gap-size;
+	font-size: $persona-font-size;
+
+	.persona-avatar {
+		display: flex;
+		align-items: center;
+		justify-content: center;
+		width: $persona-default-size;
+		height: $persona-default-size;
+		border-radius: $border-radius-rounded;
+		background-color: $persona-avatar-background-color;
+		color: $persona-avatar-font-color;
+		font-size: $persona-default-size * $persona-dimension-font-size-ratio;
+		overflow: hidden;
```

Comment:
`overflow: hidden` required?

Reply:
Yeah, it takes care of inner image. An alternative would be assigning the `border-radius` to the image itself.

Reply:
I think that's a better alternative - overflow is necessary at times, but it should still be a last resort. It affects focus outlines, popovers, etc, negatively.

Reply:
updated!